### DISCUSSION TITLE
愛上探案頻道

### DISCUSSION THREAD

**Discussion by irenesunnn** [1101-GS4719-C/Python-Community-GS4719-C#3691] [General]
坦白說我也不知道自己為什麼會迷上這類頻道😂 可能是平常工作太無聊，看這些可以滿足我的好奇心吧！而且現在的破案頻道都做得超用心，不會像以前的新聞那樣腥羶色，反而很專業！
來分享一下我的私藏名單，這些頻道我真的是一追就停不下來...（RIP我的睡眠時間）
▎重度推薦系列
1.「法醫鑑識档案」
老實說一開始看到「法醫」兩個字有點怕怕的，但這個頻道真的超專業！主持人是法醫背景，講解得很細膩，重點是不會太過驚悚。我最喜歡他解釋DNA鑑識的部分，看完整個長知識！
👉 https://youtube.com/@ForensicFiles
（我最愛的一集是講到....靠著一根頭髮就破案也太神啦！）
2.「犯罪筆記本」
這個頻道我真的是每集都追😭 超喜歡他們的敘事方式，不會太過戲劇化，就是實實在在地講案件。而且每集最後都會開放討論，讓大家一起推理，感覺自己好像在看美劇CSI一樣！
👉 https://youtube.com/@CrimeNotebook
（小聲說：這個頻道害我好幾次熬夜到三點...）
3.「心理博士時間」
這個頻道真的很不一樣！不是一味講案件，而是從犯罪心理學的角度來分析。有時候聽到分析結果都會覺得：原來人心這麼複雜！而且主持人的聲音超好聽，聽久了會上癮😆
👉 https://youtube.com/@PsychologyDoctor
我的觀看心得：
🌙 千萬不要晚上一個人看...（但我都這樣做哈哈）
🌙 如果覺得害怕，建議開著日光燈看
🌙 我都會一邊看一邊做筆記，超有偵探感！
🌙 留言區也要追，常常會有專業人士補充
說真的有時候朋友會覺得我這興趣有點怪，但我覺得這些頻道讓我學到很多！不只是案件知識，更多是人性的理解。而且現在製作都很用心，不會讓人有不舒服的感覺。
有推薦的相關頻道也歡迎跟我分享！讓我們一起當沙發偵探～

**Comment by kai1660** [comment]
推薦x檔案，will很多故事可以講，無聊需要消磨時間的話可以去看，老高也不錯，我是說還在講皇室戰爭的老高，張旭獎的微積分也講得很好，至其七七也不錯，以上都可以很好的消磨時間。
